src/nat/middleware/defense_middleware.py

```
@@ -0,0 +1,379 @@
+        """Resolve multiple JSONPath matches based on resolution strategy.
+
+        Args:
+            matches: List of JSONPath match objects
+
+        Returns:
+            List of matches based on resolution strategy
+        """
+        strategy = self.config.target_field_resolution_strategy
+
+        if strategy == "error":
+            raise ValueError(f"Multiple matches found for target_field: {self.config.target_field}")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)